README.md

```diff
@@ -1,81 +1,226 @@
     type: portal
     resource: "@SuluHeadlessBundle/Resources/config/routing_website.yml"
 ```
 
-## Usage
+This will enable a JSON API to access the **search functionality** of Sulu via `{host}/api/search` and a JSON API for 
+retrieving the **navigation contexts** of the project via `{host}/api/navigations/{contextName}`.
 
-To automatically provide the data of a page in the JSON format, you need to set the following controller in the template of your page:
+### Set the controller of you template
 
-### config/templates/pages/your-page.xml
+To provide an API for retrieving the content of a page in the JSON format, the controller of the page template
+mus be set to the `HeadlessWebsiteController` included in this bundle:
 
 ```xml
-<view>headless</view>
-<controller>Sulu\Bundle\HeadlessBundle\Controller\HeadlessWebsiteController::indexAction</controller>
+<?xml version="1.0" ?>
+<template xmlns="..." xmlns:xsi="..." xsi:schemaLocation="...">
+    <!-- ... -->
+    <controller>Sulu\Bundle\WebsiteBundle\Controller\DefaultController::indexAction</controller>
+    <!-- ... -->
+</template>
 ```
 
-### templates/headless.html.twig
+This controller will provide the **content of the page as JSON object** if the page is requested in the JSON format 
+via `{pageUrl}.json`.
 
-If you want to use the frontend setup included in this bundle, you should include the following lines in your `headless.html.twig` template to setup and start the react application:
 
-```twig
-{# ... #}
+## 💡&nbsp; Key Concepts
 
-{% block content %}
-    <div id="sulu-headless-container"></div>
+### Deliver content of pages with the HeadlessWebsiteController 
 
-    <script>window.SULU_HEADLESS_API_ENDPOINT = '{{ sulu_content_path('/api') }}';</script>
-    <script>window.SULU_HEADLESS_VIEW_DATA = {{ jsonData|raw }};</script>
+The main use-case of the SuluHeadlessBundle is **delivering the content of a page as a JSON object**. This can be 
+enabled individually per template by setting the controller of the template of the page 
+to `Sulu\Bundle\WebsiteBundle\Controller\DefaultController::indexAction`. When using the `HeadlessWebsiteController`
+as controller for a template, the content of the page is available as JSON object via `{pageUrl}.json`.
 
-    <script src="/build/build/headless/js/index.js"></script
-{% endblock %}
+Additionally to the content of the page, the JSON object returned by the `HeadlessWebsiteController` contains **meta 
+information** such as the page template and the data of the page excerpt:
+
+```json
+{
+   "id": "a5181a5a-b030-4933-b3b0-e9faf7ec756c",
+   "type": "page",
+   "template": "headless-template",
+   "content": {
+      "title": "Headless Example Page",
+      "url": "/headless-example",
+      "contacts": [
+         {
+            "id": 416,
+            "firstName": "Homer",
+            "lastName": "Simpson",
+            "fullName": "Home Simpson",
+            "title": "Dr. ",
+            "position": "Executive Vice President"
```

Comment:
his position is `Nuclear safety Inspector at the Springfield Nuclear Power Plan` :)